build-tools/README.md

```diff
@@ -85,7 +85,7 @@ eval $(docker-machine env ${KUBE_BUILD_VM})
 export KUBE_RSYNC_PORT=8370
```

Comment:
Can you change this to `8730`?   The fact it was `8370` is the typo.  (The default rsyncd port is `873`)